net/netfilter/nf_tables_api.c

```diff
@@ -4630,7 +4630,8 @@ static int nf_tables_fill_setelem(struct sk_buff *skb,
 	if (nest == NULL)
 		goto nla_put_failure;
 
-	if (nft_data_dump(skb, NFTA_SET_ELEM_KEY, nft_set_ext_key(ext),
+	if (nft_set_ext_exists(ext, NFT_SET_EXT_KEY) &&
+	    nft_data_dump(skb, NFTA_SET_ELEM_KEY, nft_set_ext_key(ext),
```

Comment:
Since I doesn't seem to show any issues with the net-filter testing I don't see a reason to remove it.

Reply:
> Since I doesn't seem to show any issues with the net-filter testing I don't see a reason to remove it.

Same as previous commit - present in rocky8_10 branch so I pulled it in.